packages/in_app_purchase/in_app_purchase_ios/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
+## 0.2.0+1
+
+* Fixes CHANGELOG for 0.2.0 (added missing information regarding the implementation of promotional offers).
+
 ## 0.2.0
 
 * BREAKING CHANGE : Refactor to handle new `PurchaseStatus` named `canceled`. This means developers
-  can distinguish between an error and user cancellation.
+  can distinguish between an error and user cancellation;
```

Comment:
Why the change from `.` to `;`? This was right before.

Reply:
since it is no longer the last element in the list. In Dutch you add a semi-colon for each item in the list except for the last item. I always thought this is the same in English.

Reply:
We've never treated the changelog as a single-sentence list gramatically, but as a series of independent sentences. Otherwise the capitalization would be wrong, and multi-sentence entries (like this one, in fact) would be very strange.